src/pages/CreateNewCourseForm.tsx

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import Box from "@mui/material/Box";
+import { BoxTitle } from "../design/typography";
+import Navbar from "../components/Navbar/Navbar";
+import { Grid, IconButton, Paper } from "@mui/material";
+import Button from '@mui/material/Button';
+import ArrowBackIosIcon from '@mui/icons-material/ArrowBackIos';
+import { useNavigate } from "react-router-dom";
+import Layout from "../components/shared/Layout";
+
+
+/*type NewCourseForm = {
+    content: number;
+    teaching:number;
+    workload: number;
+    text: string;
+  };*/
+
+
+
+
+
+export default function CreateNewCourseForm() {
+  const navigate = useNavigate();
+  function handleClick() {
+    navigate('/coursePage/');
+  }
+  /*const [ratingContent, setRatingContent] = React.useState<number>(3);
+  const [ratingTeaching, setRatingTeaching] = React.useState<number>(3);
+  const [ratingWorkload, setRatingWorkload] = React.useState<number>(3);  
+  const [textRating, setTextRating] = React.useState<string>("");
+  
+  const navigate = useNavigate();
+  function handleClick() {
+    navigate('/coursePage/');
+  }
+
+  let newCourseObj: NewCourseForm={
+    content: ratingContent,
+    teaching: ratingTeaching,
+    workload: ratingWorkload,
+    text: textRating
+  }
+  
+  const { register, handleSubmit } = useForm<NewCourseForm>();
+  
+  // eslint-disable-next-line @typescript-eslint/no-misused-promises
+  const onSubmit: SubmitHandler<NewCourseForm> = (formData) => {
+
+    newCourseObj= {"content": 5, "teaching": 4, "workload": 4, "text": "adlv"}
+    console.log(newCourseObj);
+    
+    api
+      .post("/communities" , newCourseObj)
+      .then((response) => {
+        console.log(response.data);
+        navigate("/coursePage" + response.data.id.toString());
+        return "successfull";
+      })
+      .catch((error) => {
+        console.log(error);
+        return "error";
+      });
+  };*/
```

Comment:
do we want to keep this?

Reply:
Same here, yes, since this is used for the form, which I was working on, when you asked to push my version